likeminds-chat-reactnative-integration/ChatSX/components/DeletedMessage/index.tsx

```diff
@@ -62,7 +62,7 @@ const DeletedMessage = () => {
                 : null,
             ]}
           >
-            <Text style={[styles.deletedMsg] as TextStyle}>
+            <Text style={[styles.deletedMsg] as TextStyle[]}>
```

Comment:
Why this change is required?

Reply:
As styles are multiple object bind together. In react native we pass it as an array. If it is a single object we can pass it directly.

It should like this only from the start, but since types got sorted. It raised it now.